.github/workflows/build-and-push-assets.yml

```
@@ -212,7 +202,7 @@ jobs:
       - name: Set branch environment variables
         if: ${{ github.ref_type == 'branch' }}
         run: |
-          echo "BUILT_BRANCH_NAME=${{ inputs.BUILT_BRANCH_NAME && inputs.BUILT_BRANCH_NAME || format('{0}{1}', github.ref_name, inputs.BUILT_BRANCH_SUFFIX) }}" >> $GITHUB_ENV
+          echo "BUILT_BRANCH_NAME=${{ inputs.BUILT_BRANCH_NAME && inputs.BUILT_BRANCH_NAME }}" >> $GITHUB_ENV
```

```suggestion
          echo "BUILT_BRANCH_NAME=${{ inputs.BUILT_BRANCH_NAME && inputs.BUILT_BRANCH_NAME || github.ref_name }}" >> $GITHUB_ENV
```

The thing is, we can't remove support for the legacy (or, to be more precise, an original) approach to push compiled to the **current** branch. If we really want this, we should make the `BUILT_BRANCH_NAME` input required. It should be prepended deprecation step:

```yaml
- name: Deprecation warning
        if: ${{ inputs.BUILT_BRANCH_NAME = '' }}
        run: |
          echo '::warning::The BUILT_BRANCH_NAME will be required soon. If you push compiled assets to the current, branch you should use ${{ github.ref_name }} value.'
``` 

But the main question is what we consider the upstream approach :smile: